sign-and-combine-ecdsa/nodejs/README.md

```diff
@@ -0,0 +1,51 @@
+# `signAndCombineEcdsa` Lit Action Code Example
+
+This code demonstrates how to use the Lit SDK to implement the `signAndCombineEcdsa` method in a Lit Action. This method combines the signature shares of your PKP from each node onto a single node within a Lit Action, meaning they remain in Lit's Trusted Execution Environment (TEE) and are not exposed to the client.
```

Comment:
this uses the Lit Actions SDK right?

Reply:
also, i like the link this section when we reference the TEE: https://developer.litprotocol.com/resources/how-it-works#sealed-and-confidential-hardware

so new users have some more context on what we are talking about

Reply:
Sounds good, I've made these changes